src/rpdk/core/jsonutils/resolver.py

```diff
@@ -104,7 +113,7 @@ def _schema_to_lang_type(self, property_schema):
             return self._get_array_lang_type(property_schema)
         if schema_type == "object":
             return self._get_object_lang_type(property_schema)
-        return self._get_primitive_lang_type(schema_type)
+        return self._get_primitive_lang_type(convert_list_to_object(schema_type))
```

Comment:
It's more like hacking into the primitive type, but actually we should have a different type. e.g. 

```
if isinstance(schema_type, list)
       return self._get_multiple_lang_type(property_schema)

def _get_multiple_lang_type(self, property_schema):
 return ResolvedType(ContainerType.MULTIPLE, property_schema)

```

Reply:
done